rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.py

```diff
@@ -11,15 +11,11 @@ def rule(event):
 
 def title(event):
     return (
-        "Startup script for instance"
-        f" [{deep_get(event, 'requestParameters', 'instanceId')}] "
-        "was modified."
+        f"[{deep_get(event,'userIdentity','arn')}] "
+        "modified the startup script for "
+        f" [{deep_get(event, 'requestParameters', 'instanceId')}]."
     )
 
 
```

Comment:
Why did we remove the dedup on this?

Reply:
My idea was that the title would be a more granular dedup since it included the user and instance but on second thought maybe having a shorter dedup string with just the instanceid stored is more efficient.

Reply:
ah, dedup defaults to `title` .. that's probably fine, though historically my understanding is that instanceIDs are unique to the following domain
* AWS Account ID +  AWS Region 

The AWS Docs on it aren't as clear as my mental picture, so maybe I'm off https://github.com/awsdocs/amazon-ec2-user-guide/blob/master/doc_source/resource-ids.md#resource-ids